edx_proctoring/tests/test_views.py

```diff
@@ -2659,3 +2660,41 @@ def test_launch_for_configuration_dashboard(self):
                                 kwargs={'course_id': course_id, 'exam_id': exam_id})
         response = self.client.get(dashboard_url, {'config': 'true'})
         self.assertRedirects(response, expected_url, fetch_redirect_response=False)
+
+
+class TestBackendUserDeletion(LoggedInTestCase):
+    """
+    Tests for deleting user data from backends
+    """
+    def setUp(self):
+        super(TestBackendUserDeletion, self).setUp()
+        self.user.is_staff = True
+        self.user.save()
+        self.second_user = User(username='tester2', email='tester2@test.com')
+        self.second_user.save()
+        self.client.login_user(self.user)
+
+    def test_can_delete_user(self):
+        deletion_url = reverse('edx_proctoring:backend_user_deletion_api', kwargs={'user_id': self.second_user.id})
+
+        response = self.client.delete(deletion_url)
+        assert response.status_code == 200
+        data = response.json()
+        assert data == {'test': True}
+        test_backend = get_backend_provider(name='test')
+        assert test_backend.last_retire_user is not None
```

Comment:
The rest of this file's test classes use `self.assetStuff` calls; is there motivation to break from that for these tests?

Reply:
It turns out that plain asserts work better than the various assert methods when run with py.test (py.test will print more helpful error messages). I heard recently that it was recommended to switch to plain asserts across the platform, although nobody is going to go back to refactor all of the old tests.